Source/Core/DolphinQt/MainWindow.cpp

```diff
@@ -142,6 +148,13 @@ QString DMainWindow::ShowFileDialog()
 		.arg(SL("*.gcm *.iso *.ciso *.gcz *.wbfs *.elf *.dol *.dff *.tmd *.wad")));
 }
 
+QString DMainWindow::ShowFolderDialog()
+{
+	return QFileDialog::getExistingDirectory(this, tr("Browse for a directory to add"),
+                                             QDir::homePath(),
+                                             QFileDialog::ShowDirsOnly);
+}
```

Comment:
I'm fairly certain those two lines are being aligned with spaces. I checked with a hexeditor to be 100% sure:
![hexedit](https://i.imgur.com/3xdYr9y.png)